invenio_app_ils/circulation/views.py

```diff
@@ -73,6 +77,17 @@ def create_circulation_blueprint(app):
         "/circulation/loans/create", view_func=loan_create, methods=["POST"]
     )
 
+    loan_mail = LoanMailResource.as_view(
+        LoanMailResource.view_name.format(CIRCULATION_LOAN_PID_TYPE),
+        serializers=serializers,
```

Comment:
If merged after my PR, when you rebase, it is better to add `default_media_type` here:
https://github.com/inveniosoftware/invenio-app-ils/pull/446/files#diff-f90fd9b30de705c5af2a29d136fedad0R50

Reply:
I have added it already.